application-licensing-licensor/application-licensing-licensor-api/src/main/java/com/xwiki/licensing/internal/DefaultLicensingConfiguration.java

```diff
@@ -102,4 +110,34 @@ public File getLocalStorePath()
         }
     }
 
+    @Override
+    public String getStoreTrialURL()
+    {
+        return this.storeConfig.getProperty("storeTrialURL");
```

Comment:
@mflorea Should we provide here a default value, like "http://url", to avoid an exception while building the trial url here https://github.com/xwikisas/application-licensing/pull/94/files#diff-cfa828b50efb9640c7d353996f2d315d4074ef6b5835174fc0cdfc8560e6df08R113 ?
Same for getStoreUpdateURL()

Reply:
I don't think there is a good default value here (and same for ``getStoreUpdateURL()``) so I think the code that calls these methods should check if the return value is empty/null and don't do anything if it's the case (e.g. don't get a license automatically).